tenant/sunbird/newLandingPage.html

```diff
@@ -0,0 +1,919 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
@Alfaj2302 - Update the MR Title in standard format

Reply:
we don't have tracker link yet

Reply:
@Ekta29yadav Add task in the PMS and use it

Reply:
I dont have acess yet

Reply:
@komalm done